src/pages/Bets/Bets_Logic.tsx

```diff
@@ -8,6 +8,15 @@ const getMockBets = () => {
   return Array.from({ length: 5 }, () => undefined);
 };
 
+const formatAmount = (amountStr: string) => {
+  const amount = parseFloat(amountStr);
+  if (amount < 0.0001) return "0.0001<";
```

Comment:
This it would more often be written this way? Not certain
```suggestion
  if (amount < 0.0001) return "<0.0001";
```

Reply:
will do